modules/host/contractmanager/sectorupdate.go

```diff
@@ -10,6 +10,12 @@ import (
 	"github.com/NebulousLabs/Sia/modules"
 )
 
+const (
+	// maxParallelThreads is the maximum number of threads updating
+	// sector counters on disk in AddSectorBatch.
+	maxParallelThreads = 100
+)
```

Comment:
belongs in consts.go, but will need a name like `maxAddSectorBatchThreads`

Reply:
I guess if we're being pedantic, it needs to say `maxAddSectorBatchGoroutines`

Reply:
we use `Threads` elsewhere in the codebase, so might as well stick with that